service/src/main/java/uk/gov/hmcts/reform/fpl/service/ChildrenService.java

```diff
@@ -80,6 +81,48 @@ public boolean allChildrenHaveFinalOrder(List<Element<Child>> children) {
         return children;
     }
 
+    public List<Element<Child>> updateFinalOrderIssued(CaseData caseData) {
+        return updateFinalOrderIssued(
+            caseData.getManageOrdersEventData(),
+            caseData.getAllChildren(),
+            caseData.getOrderAppliesToAllChildren(),
+            caseData.getChildSelector(),
+            caseData.getRemainingChildIndex()
+        );
+    }
+
+
+    private List<Element<Child>> updateFinalOrderIssued(ManageOrdersEventData manageOrdersEventData,
+                                                       List<Element<Child>> children, String orderAppliesToAllChildren,
```

Comment:
Maybe we could have a boolean instead of the String orderAppliesToAllChildren parameter.

Reply:
Had a look but the getOrderAppliesToAllChildren function in CaseData returns a String so probably easier to compare using an enum rather than changing to a boolean to pass into this function